Composer/packages/client/src/CreationFlow/LocationBrowser/FileSelector.tsx

```diff
@@ -39,116 +40,108 @@ interface FileSelectorProps {
   checkShowItem: (file: File) => boolean;
 }
 
+type SortState = {
+  key: string;
+  descending: boolean;
+};
+
+const _renderIcon = (file: File) => {
+  const iconName = getFileIconName(file);
+  if (iconName === FileTypes.FOLDER) {
+    return <Icon style={{ fontSize: '16px' }} iconName="OpenFolderHorizontal" />;
+  } else if (iconName === FileTypes.BOT) {
+    return <Icon style={{ fontSize: '16px' }} iconName="Robot" />;
+  } else if (iconName === FileTypes.UNKNOWN) {
+    return <Icon style={{ fontSize: '16px' }} iconName="Page" />;
+  }
+  // fallback for other possible file types
+  const url = `https://static2.sharepointonline.com/files/fabric/assets/brand-icons/document/svg/${iconName}_16x1.svg`;
```

Comment:
it looks like svg files are only stored on sharepoint online for `.md` files. Otherwise I see them saved into the repository. 
![image](https://user-images.githubusercontent.com/14900841/79147526-ea611580-7d78-11ea-8b54-c4a9d2c17094.png)

I recommend doing this same for this.

Reply:
That fallback code wasn't mine either; it was something I moved as part of the "what is this file icon" logic that was farther down. It smells a little funny to me, too, but we can fix that in a later PR if it's worth doing.

Reply:
Thanks for the reply! sounds good to me.